apps/src/templates/lessonOverview/activities/ActivitySection.jsx

```diff
@@ -73,29 +57,10 @@ export default class ActivitySection extends Component {
             </span>
           )}
         </h3>
-        <div
-          style={{
-            ...styles.activitySection,
-            ...(sectionHasTips && {position: 'relative', left: -30})
-          }}
-        >
-          {sectionHasTips && (
-            <div style={styles.tipIcons}>
-              {section.tips.map((tip, index) => {
-                return (
-                  <FontAwesome
```

Comment:
Oh wow I didn't realize we were removing the icons from the left side.

Reply:
That's what it looks to me from the spec. Happy to put them back if I misunderstood.

Reply:
I think based on the spec it is right. I just hadn't internalized that change